bin/ycsb

```diff
@@ -107,9 +108,15 @@ command = COMMANDS[sys.argv[1]]["command"]
 database = sys.argv[2]
 db_classname = DATABASES[database]
 options = sys.argv[3:]
+java_home = os.environ["JAVA_HOME"]
```

Comment:
great. presuming you go forward with the rebase, please edit the commit message for this part of the changes to note "fixes #252"